.github/coderabbit.yaml

```
@@ -0,0 +1,60 @@
+# CodeRabbit設定ファイル
+# https://docs.coderabbit.ai/
+
+# 基本設定
+language: ja                      # 日本語でレビュー
+
+# レビュー設定
+reviews:
+  profile: "chill"               # レビューのスタイル (chill, assertive, etc.)
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)